### ISSUE TITLE
[Package Issue]: Microsoft.PowerBI

### ISSUE THREAD

**Issue by TheStingPilot** [microsoft/winget-pkgs#150922]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

The installation quits with the error:
`Installer hash does not match; this cannot be overridden when running as admin`


### Steps to reproduce

Run as admin:

`"C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.22.10861.0_x64__8wekyb3d8bbwe\winget.exe" install --exact --id Microsoft.PowerBI --silent --accept-package-agreements --accept-source-agreements --scope machine --source winget`

### Actual behavior

```
[Installation] ::    - 
   \ 
   | 
   / 
                                                                                                                        
  ++                              1024 KB / 10.1 MB
  +++++                           2.00 MB / 10.1 MB
  ++++++++                        3.00 MB / 10.1 MB
  +++++++++++                     4.00 MB / 10.1 MB
  ++++++++++++++                  5.00 MB / 10.1 MB
  +++++++++++++++++               6.00 MB / 10.1 MB
  ++++++++++++++++++++            7.00 MB / 10.1 MB
  +++++++++++++++++++++++         8.00 MB / 10.1 MB
  ++++++++++++++++++++++++++      9.00 MB / 10.1 MB
  +++++++++++++++++++++++++++++   10.0 MB / 10.1 MB
  ++++++++++++++++++++++++++++++  10.1 MB / 10.1 MB
                                                                                                                        
   - 
                                                                                                                        
Found Microsoft PowerBI Desktop [Microsoft.PowerBI] Version 2.128.952.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Agreements for Microsoft PowerBI Desktop [Microsoft.PowerBI] Version 2.128.952.0
Version: 2.128.952.0
Publisher: Microsoft Corporation
Publisher Url: https://www.microsoft.com/en-us
Publisher Support Url: https://support.microsoft.com/en-us
Author: Microsoft Corporation
Homepage: https://powerbi.microsoft.com
License: Proprietary
License Url: https://powerbi.microsoft.com/en-us/desktop-eula/
Privacy Url: https://privacy.microsoft.com/en-us/privacystatement
Copyright: Copyright (c) Microsoft Corporation. All rights reserved.
Copyright Url: https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx
Agreements:
  End User License Agreement (EULA): https://powerbi.microsoft.com/en-us/desktop-eula/
Downloading https://download.microsoft.com/download/8/8/0/880BCA75-79DD-466A-927D-1ABF1F5454B0/PBIDesktopSetup_x64.exe
   - 
   \ 
                                                                                                                        
                                                                                                                        
                                  0.00 B /  487 MB
  +++                             49.0 MB /  487 MB
  +++++++                          114 MB /  487 MB
  ++++++++++                       173 MB /  487 MB
  ++++++++++++++                   238 MB /  487 MB
  ++++++++++++++++++               294 MB /  487 MB
  +++++++++++++++++++++            355 MB /  487 MB
  ++++++++++++++++++++++++         397 MB /  487 MB
  ++++++++++++++++++++++++++       437 MB /  487 MB
  +++++++++++++++++++++++++++++    477 MB /  487 MB
  ++++++++++++++++++++++++++++++   487 MB /  487 MB
Installer hash does not match; this cannot be overridden when running as admin
```


### Expected behavior

A correct installed PowerBI Desktop, installed with admin rights.

### Environment

```raw
WinGet version: v1.7.10861
```


### Screenshots and Logs

n/a

**Comment by vikingnope** [comment]
This will be fixed with either: https://github.com/microsoft/winget-pkgs/pull/150882 OR: https://github.com/microsoft/winget-pkgs/pull/150865

**Comment by stephengillie** [comment]
> This will be fixed with either: #150882 OR: #150865
